test/integrations/sources/segment/data.ts

```diff
@@ -0,0 +1,239 @@
+import utils from '../../../../src/v0/util';
+
+const defaultMockFns = () => {
+  jest.spyOn(utils, 'generateUUID').mockReturnValue('97fcd7b2-cc24-47d7-b776-057b7b199513');
+};
+
+export const data = [
```

Comment:
Should we add type checking here as well ?

```suggestion
export const data : TestCaseData[] = [
```